packages/config/src/layer2s/canvasconnect.ts

```diff
@@ -0,0 +1,146 @@
+import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/shared'
+
+import {
+  CONTRACTS,
+  DATA_AVAILABILITY,
+  EXITS,
+  FORCE_TRANSACTIONS,
+  makeBridgeCompatible,
+  NEW_CRYPTOGRAPHY,
+  NUGGETS,
+  OPERATOR,
+  RISK_VIEW,
+  SHARP_VERIFIER_CONTRACT,
+  STATE_CORRECTNESS,
+} from './common'
+import { Layer2 } from './types'
+
+export const canvasconnect: Layer2 = {
+  type: 'layer2',
+  id: ProjectId('canvasconnect'),
+  display: {
+    name: 'Canvas Connect',
+    slug: 'canvasconnect',
+    description:
+      'Canvas Connect is a privacy focused Layer 2 ZK network built for Finance and the Digital Economy. Connect offers an API first model that supports easy integration with no solidity skills required. Capital Markets Applications are built on the network that enable the Transfer, Trade & Investment of Tokenised Real World Assets, Stablecoins, CBDCs & Digital Assets with assurances of Privacy, Low Cost and High Speed, all secured by Ethereum.',
+    purpose:
+      'Transfer, Trade & Investment of Tokenised Real World Assets, Stablecoins, CBDCs & Digital Assets with assurances of Privacy, Low Cost and High Speed, all secured by Ethereum.',
+    links: {
+      websites: ['https://canvas.co/'],
+      apps: [],
+      documentation: ['https://docs.starkware.co/starkex/index.html'],
+      explorers: [],
+      repositories: ['https://github.com/starkware-libs/starkex-contracts'],
+      socialMedia: [
+        'https://twitter.com/canvas_defi',
+        'https://www.youtube.com/@canvas_defi',
+        'https://canvasdefi.medium.com/',
+        'https://www.linkedin.com/company/canvasblockchaingroup',
+        'https://www.canvas.co/content',
+      ],
+    },
+    activityDataSource: 'Closed API',
+  },
+  config: {
+    escrows: [],
+  },
```

Comment:
I'm slightly confused by this. The project will always have 0 TVL unless we manually add the L2 native tokens. Also, please explain, how would the DEFI pooling work without any L1 assets.